src/LibFLExBridge-ChorusPluginTests/Integration/UpdateMetaDataCacheTests.cs

```diff
@@ -333,6 +333,16 @@ public void MetaDataCacheIsUpdated()
 
 			// 7000071: No actual model change.
 			CheckNoModelChangesUpgrade(mdc, fileHandler, 7000071);
+
+			// 7000072:
+			// Make sure prior model is expected.
+			//Removing ReversalEntries property from LexSense from owning collection to owning sequence.
```

Comment:
Remove part of the comment leaving: //Removing ReversalEntries property from LexSense